debian/cart.install

```diff
@@ -0,0 +1,6 @@
+usr/bin/cart_ctl
+usr/bin/crt_launch
+usr/bin/*-cart_ctl
+usr/bin/*-crt_launch
+usr/lib/*/cart/utils/*
+usr/etc/*
```

Comment:
self_test should be part of cart.install and not cart-tests, as it is a utility to test performance of servers and not really a 'test' on its own.

Reply:
Change will be in the next update.  However I would recommend having a binary name that is clearly associated with cart instead of just "self_test".